hera_cal/lst_stack/tests/conftest.py

```diff
@@ -0,0 +1,162 @@
+import pytest
```

Comment:
I don't think I'm going to have a chance to review the tests this round... @tyler-a-cox can you take a quick look at them?

Reply:
@jsdillon yes, will take a look now